src/EnergyPlus/UnitarySystem.cc

```diff
@@ -7884,16 +7860,19 @@ namespace UnitarySystems {
     {
         std::string cCurrentModuleObject = "ZoneHVAC:PackagedTerminalAirConditioner";
         SysType sysTypeNum = SysType::PackagedAC;
+        DataZoneEquipment::ZoneEquip zoneEqTypeNum = DataZoneEquipment::ZoneEquip::Invalid;
```

Comment:
New comptype for zone equipment unitary systems. Would be nice if these worked off a common enum list, but they don't yet, maybe never, not sure what's the best plan.

Reply:
These changes don't appear to be a conflict issue with #9627, we'll see.